include/usb_cam/utils.hpp

```diff
@@ -146,31 +151,41 @@ inline std::map<std::string, v4l2_capability> available_devices()
   // Initialize vector of device strings to fill in
   std::map<std::string, v4l2_capability> v4l2_devices;
 
-  // V4L2 spec says there can only be a maximum of 64 devices
-  const uint8_t MAX_DEVICES = 64;
+  // Get a list of all v4l2 devices from /sys/class/video4linux
```

Comment:
@firesurfer the list in `/sys/class/video4linux` should include all valid v4l2 devices I think...even if they are `/dev/media` devices